app/views/boxes/_form.haml

```diff
@@ -0,0 +1,54 @@
+= form_for(@box) do |f|
+
+  = validation_errors @box
+
+  .row
+    .col
+      .row.form-field
+        .col.pe-3
+          = f.label :uuid
+        .col
+          .value= @box.uuid
+
+      .row.form-field
+        .col.pe-3
+          = f.label :purpose
+        .col
+          = react_component "CdxSelectSectionToggler", { name: "box[purpose]", items: Box.purposes.map { |v| { label: v, value: v } }, value: @box.purpose, submit: "#btn-save" }
+
+      #LOD
+        .row.form-field
+          .col.pe-3
+            = label_tag "box_batch_numbers_0", "Batch"
+          .col
+            = text_field_tag "box[batch_numbers][0]", "", id: "box_batch_numbers_0", disabled: true, placeholder: "Enter batch id"
+
+      #Variants
+        - 1.upto(6) do |index|
+          .row.form-field
+            .col.pe-3
+              = label_tag "box_batch_numbers_#{index}", "Variant #{index}"
+            .col
+              = text_field_tag "box[batch_numbers][#{index}]", "", id: "box_batch_numbers_#{index}", disabled: true, placeholder: "Enter batch id"
+
+      #Challenge
+        .row.form-field
+          .col.pe-3
+            = label_tag "box_batch_numbers_0", "Virus"
+          .col
+            = text_field_tag "box[batch_numbers][id]", "", id: "box_batch_numbers_0", disabled: true, placeholder: "Enter batch id"
```

Comment:
Why `id` instead of `0`?

```suggestion
            = text_field_tag "box[batch_numbers][0]", "", id: "box_batch_numbers_0", disabled: true, placeholder: "Enter batch id"
```